### PR TITLE
Swerve go BRRRR

### PR DESCRIPTION
**Pull request by jeffaburke**
Making the swerve code work somewhat well

### CODE REVIEW

File: src/main/java/frc/robot/maps/GastonMap.java [hunk 1]
```diff
@@ -30,31 +28,31 @@ public SwerveDriveMap getSwerveDriveMap() {
         // All Distances are in Meters
         // Front Left Module
         final CANCoder encoderFL = new CANCoder(1);
-        encoderFL.configMagnetOffset(0); // TODO Get Magnet Offset
+        encoderFL.configMagnetOffset(-195.381);
```
Comment (msoucy) [inline src/main/java/frc/robot/maps/GastonMap.java:32]: Please look [here](https://www.chiefdelphi.com/t/official-sds-mk3-mk4-code/397109/96) for some info that might help with this.

File: src/main/java/frc/robot/subsystems/Drive.java [hunk 2]
```diff
@@ -52,7 +53,9 @@ private void updateSwerveSpeedAngle(final DoubleSupplier translateX, final Doubl
     final double translateXSpeed = deadband.applyAsDouble(translateX.getAsDouble()) * maxDriveSpeedMetersPerSecond;
     final double translateYSpeed = deadband.applyAsDouble(translateY.getAsDouble()) * maxDriveSpeedMetersPerSecond;
     final double rotationSpeed = deadband.applyAsDouble(rotation.getAsDouble()) * maxRotationRadiansPerSecond;
-
+    SmartDashboard.putNumber("translate y", translateYSpeed);
+    SmartDashboard.putNumber("translate x", translateXSpeed);
+    SmartDashboard.putNumber("retation speed", rotationSpeed);
```
Comment (msoucy) [inline src/main/java/frc/robot/subsystems/Drive.java]: Please fix capitalization/spelling... or just remove.

**Review by msoucy — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/maps/GastonMap.java (hunk 1 above)
Comment (jeffaburke) [reply src/main/java/frc/robot/maps/GastonMap.java:32]: What do you mean by this?

Comment (msoucy) [reply src/main/java/frc/robot/maps/GastonMap.java:32]: Seems to not be as useful as I was expecting - this is fine

**Review by msoucy — APPROVED** [review]

**Pull request merged**